### PR TITLE
Add localization support (Work in Progress)

### PR DESCRIPTION
**Pull request by Mahas1**
When this is done, this will enable users to pick languages to use this software in, and will enable contributors to easily add their own language to the language list.

### CODE REVIEW

**Review by kernel-dev — APPROVED** [review]

**Review by kernel-dev — DISMISSED** [review]

File: src/cli/ui.py [hunk 1]
```diff
@@ -70,7 +69,8 @@ def handle_cmd(self, options=[]):
                 title()
                 data = option[2]()
 
-                print(data if data else "Successfully executed.\n")
+                print(data if data else
```
Comment (kernel-dev) [inline src/cli/ui.py:72]: Are we checking here if `data` is an empty string? If so, I believe it's sufficient to just do: `data or <expr>`

File: src/cli/ui.py [hunk 2]
```diff
@@ -91,14 +91,15 @@ def toggle_data(self):
...
-            ("G", "GPU"),
-            ("B", "Motherboard" if system().lower() != "darwin" else "Vendor"),
-            ("M", "Memory"),
-            ("N", "Network"),
-            ("A", "Audio"),
-            ("I", "Input"),
-            ("S", "Storage"),
+            ("C", self.langparser.parse_message("src-cli-ui-cpu")),
+            ("G", self.langparser.parse_message("src-cli-ui-gpu")),
+            ("B", self.langparser.parse_message("src-cli-ui-motherboard") if system().lower() != "darwin"
```
Comment (kernel-dev) [inline src/cli/ui.py:96]: We could alternatively rename this to `baseboard`, as I believe that's more of a unified terminology, so there wouldn't be a need to have this conditional branching here.

**Comment by kernel-dev** [comment]
I'm just putting this here in case you're unaware, but sometimes, it'll exit out of the application after executing a command.

https://github.com/KernelWanderers/OCSysInfo/assets/61620747/1e294f04-6624-4932-862c-edee63571db2

File: src/cli/ui.py (hunk 2 above)
Comment (Mahas1) [reply src/cli/ui.py:96]: will keep this in mind.

**Pull request merged**